apps/condo/domains/notification/constants/constants.js

```diff
@@ -52,6 +54,8 @@ const MESSAGE_TYPES = [
     TICKET_COMMENT_ADDED_TYPE,
     BILLING_RECEIPT_AVAILABLE_MANUAL_TYPE,
     BILLING_RECEIPT_AVAILABLE_NO_ACCOUNT_MANUAL_TYPE,
+    VERIFICATION_DATE_REMINDER_30_DAYS_TYPE,
```

Comment:
it's not enough to send the message! You need to use one type and use the remaining time as a parameter.

Reply:
Fixed